snapshotter/deployment/roles/snapshotter/tasks/configure_snapshotter.yml

```diff
@@ -9,6 +9,11 @@
     path: /var/miner_config
     state: directory
 
+- name: Create miner logs directory
```

Comment:
Actually, this is for the snapshotter, not miner. @vpetersson can you point me to the miner install bits? Or was that done manually?

Reply:
Updated the PR. Hope that adds more clarity.